tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["es6"],
+    "noImplicitAny": true,
```

Comment:
This (`noImplicitAny`) to `strictNullChecks` are covered by `strict`

Reply:
good idea, refactor to use `strict`